spec/models/organization_spec.rb

```diff
@@ -352,6 +353,18 @@
     end
   end
 
+  describe 'default invite and approve value' do
+    it 'returns false when not set' do
+      expect(build(:organization)).to_not be_single_step_invite_and_approve
+    end
+
+    it 'updates to true' do
+      org = create(:organization)
+      org.update(single_step_invite_and_approve: true)
+      expect(org.single_step_invite_and_approve).to eq(true)
```

Comment:
This test doesn't actually test anything other than the standard Rails `update` method. It can be removed.